bookwyrm/templates/rss/edition.html

```diff
@@ -0,0 +1,4 @@
+{{obj.title}} by {{obj.author_text}}
```

Comment:
I believe this should be wrapped with `{% blocktrans %} … {% endblocktrans %}`.

I still want to improve/review this format a bit, but I say we merge the PR with the two suggestions above, and do a follow-up (unless you prefer one extra round here.

Basically what I want to review is the ISBN labels: not sure if we want to show both if both are available?; and check whether newlines are needed somewhere, for readability in RSS readers.)

Reply:
I can add the wrapping no problem. I would like to merge soon so I can see some incremental value. The i8n effort I think was merged after I started this PR and I want to merge before folks start any new initiatives!

Reply:
wrapping is done in 1d8bd2be892dde3079915a6397e28edeba1c545f

Reply:
in forms we have multiple places to put the isbn, both 10 and 13 - but in the templates for books and editions I find that we handle book identifiers in https://github.com/bookwyrm-social/bookwyrm/blob/main/bookwyrm/templates/book/book_identifiers.html 
In that, we ONLY display ISBN_13. I'm fine to amend this to show each identifier as it exists similar to the book_identifiers helper

Reply:
amended to have no ISBN10  but the other identifiers in book_identifiers in 09d857e6fbf731283d4568f1ac4354e0120722ca

Reply:
this is excellent, thank you <3